app/views/denounces/_form.html.erb

```diff
@@ -0,0 +1,62 @@
+
+<%= javascript_pack_tag 'addressSelectors.js' %>
+<%= bootstrap_form_with(model: denounce, layout: :horizontal, label_col: "col-12", control_col: "col-12", inline_errors: true) do |form| %>
+  <div class="card-body mx-4 my-4">
+    <div class="row">
+      <div class="col-12 col-lg-6">
+        <h3>Denounce Information</h3>
+        <%= form.datetime_select :event_date %>
+        <%= form.select :area, Denounce.areas.keys %>
+        <%= form.text_field :title %>
+        <%= form.text_field :description %>
+        <%= form.check_box :anonymouse, label: "Make it anonymous" %>
+      </div>
+
+      <div class="col-12 col-lg-6">
+        <h3>Address</h3>
+        <%= form.fields_for :address, denounce.address || Address.new do |form_field| %>
```

Comment:
I think the address assignment could be in the controller

Reply:
> I think the address assignment could be in the controller

See : https://github.com/BC-MAR-21-ROR/proyecto-denuncia-ciudadana-proyecto-team02/pull/16/commits/86e10e129e189cb30929a514c2c86d612b081998